templates/DatasetReview/submit.html.twig

```diff
@@ -73,7 +73,7 @@
                     <b>DOI:</b>
                 </td>
                 <td>
-                    {{ DatasetSubmission.Dataset.Doi|default }}
+                    {{ DatasetSubmission.Dataset.Doi.getDoi|default }}
```

Comment:
get is not needed. Twig automatically tries to find the function. get,has,is are some examples.

Reply:
Yes, I don't know why i put that get there . Will change it